api/apps/api/src/app.module.ts

```diff
@@ -65,6 +66,7 @@ import { BlmValuesModule } from '@marxan-api/modules/blm';
     ScenarioSpecificationModule,
     PublishedProjectModule,
     BlmValuesModule,
+    ProjectAclModule,
```

Comment:
`project-acl` module isn't indented to be used directly anywhere (also take look at the import path).